packages/flutter/test/foundation/assertions_test.dart

```diff
@@ -352,4 +352,80 @@ void main() {
       expect(summary.value, equals(<String>['Invalid argument(s) (myArgument): Must not be null']));
     }
   });
+
+  test('Identifies user fault', () {
+    // User fault because they called `new Text(null)` from their own code.
+    final StackTrace stack = StackTrace.fromString('''#0      _AssertionError._doThrowNew (dart:core-patch/errors_patch.dart:42:39)
```

Comment:
The first line could also be on a newline for readability (the new line followin `'''` in multiline string is ignored)